rdflib/graph.py

```diff
@@ -1070,13 +1072,24 @@ def parse(
         )
         if format is None:
             format = source.content_type
+        assumed_xml = False
         if format is None:
-            # raise Exception("Could not determine format for %r. You can" + \
-            # "expicitly specify one with the format argument." % source)
-            format = "application/rdf+xml"
+            try:
+                from rdflib.util import guess_format  # local import avoids circular dependency
```

Comment:
It is probably better to just move the import to the end of the file but keep it at the top level in this situation.

Reply:
I refactored to use an absolute module import at the top, as per [this suggestion](https://stackoverflow.com/a/37126790/1820042). I changed two imports in `rdflib.util` to accommodate this. This seems like an improvement.